Common/Product/SharedProject/ProjectNode.cs

```diff
@@ -1593,6 +1597,16 @@ internal enum EventTriggering {
             }
         }
 
+        private void LogProjectLoadTelemetry(string eventName) {
+            DataPointCollection properties = new DataPointCollection();
+            // Enclose guid in braces
+            properties.Add(TelemetryProperties.ProjectGuid, this.ProjectIDGuid.ToString("B"));
+            properties.Add(TelemetryProperties.ProjectFlavorGuid, this.ProjectGuid.ToString("B"));
+            properties.Add(TelemetryProperties.ProjectType, this.ProjectType);
+
+            NodejsPackage.Instance.TelemetryLogger.ReportEvent(eventName, properties);

```

Comment:
Removed this part as mentioned in above comment. Also, existing VS telemetry has ActiveProject property to track projectid of current project when event was sent. From projectGuid we can get these extra properties on our data backend by simple joins.